search_action_domain/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "Search Action Domain",
+    "summary": "Send a list of server actions via context to join the domains on the returned actions to a search.",
+    "author": "Vauxoo",
+    "website": "https://www.vauxoo.com",
+    "license": "LGPL-3",
+    "category": "Base/Base",
```

Comment:
I'd use "Extra Tools", as most modules in server-ux

Reply:
Done!